app/views/chess_boards/show.html.erb

```diff
@@ -12,37 +12,72 @@
   </head>
   <body>
     <br><br><br>
-
+    
     <div class="chessboard">
 
         <% (0..7).each do |row| %>
           <div id="row">
-         <% (0..7).each do |column| %>
+        <% (0..7).each do |column| %>
            <% if (row % 2 == 0) %>
              <% if (column % 2 == 0) %>
-                <div class="white"></div>
-             <% else %>
-                <div class="black"></div>
-             <% end %>
-          <% else %>
-             <% if (column % 2 == 1) %>
-                <div class="white"></div>
-             <% else %>
-                <div class="black"></div>
-             <% end %>
-           <% end %>
-         <% end %>
-          </div>
-        <% end %>
 
-    </div>
+                <div class="white">
+                <% @pieces.each do |piece| %>
```

Comment:
Sounds good - tried and I am not doing it right. I am substituting that one line of code - the loop with `<% @pieces.get_piece(x, y) %>`. I don't think this is right because I don't think I am calling the Piece Class first. Got errors. I looked at examples of how to call a model method in a view and it seemed almost too straightforward. Do I need to put them in the 4 divs or can the line of (correct) code be placed in one spot on the page?